bin/install-qa-check.d/60openrc

```diff
@@ -33,6 +33,21 @@ openrc_check() {
 			done
 		done
 	fi
+
+	# Make sure deprecated #!/sbin/runscript isn't being used:
+	if [[ ! -d ${ED}/etc/init.d ]] ; then
+		continue
+	fi
+
+	for x in ${ED}/etc/init.d/* ; do
+		if [[ -f ${ED}/etc/init.d/* ]] ; then
+			pass
+		fi
+
+		if [[ "$(head -n1 "${x}")" =~ "#!/sbin/runscript" ]] ; then
```

Comment:
I went for consistency with the earlier runscript header check (and IMHO it's less readable for a single if statement versus a loop statement.